feat: update authorization documentation (#150)
meshcloud/meshcloud-docs: docs/administration.index.md

- **meshAdmin**: It's basically the account with administration rights. Has full access to all functionality and can manage users of the
  administrator/partner account.
- **meshAdmin Employee**: Has full access to all administration functionality, but **cannot** manage users of the administrator/partner account.
- **Platform Operator**: An operator of a cloud platform, that is managed by the meshcloud platform. This role can perform tasks related to
  platform operations in the administration area.
- **Ops Support**: Not all Ops operations must be executed by Platform Operators. E.g. setting quotas is a task that can also be done by
  a separate Ops Support team.
- **Onboarding Support**: A support team in place to help users who want to sign up may also need access to some administration
  functionality which is granted by this role.
- **Controller**: A Controller has access to billing and usage information of customer projects.

> See the [meshCustomer](meshcloud.customer.md) documentation for details about how you can manage the roles of your users. This set of roles is easily extendable, so individual access requirements of your organization can be implemented.

Reviewer: ```suggestion
See the [meshCustomer](meshcloud.customer.md) documentation for details about how you can manage the roles of your users. This set of roles is easily extendable by adding roles to reflect individual access requirements of your organization.
```
